File: app/src/main/java/com/example/configurers/RemoteConfig.java
package com.example.configurers;

import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings;

import com.example.BuildConfig;

public final class RemoteConfig {
  public static void setup() {
    final FirebaseRemoteConfigSettings remoteConfigSettings =
      new FirebaseRemoteConfigSettings
        .Builder()
        .setDeveloperModeEnabled(BuildConfig.DEBUG)
        .build();

    FirebaseRemoteConfig
      .getInstance()
      .setConfigSettings(remoteConfigSettings);
  }
}

File: app/src/main/kotlin/com/example/configurers/RemoteConfig.kt
package com.example.configurers

import com.example.BuildConfig
import com.google.firebase.remoteconfig.FirebaseRemoteConfig
import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings

object RemoteConfig {
  fun setup() {
    val remoteConfigSettings = FirebaseRemoteConfigSettings.Builder()
      .setDeveloperModeEnabled(BuildConfig.DEBUG)
      .build()

    FirebaseRemoteConfig
      .getInstance()
      .setConfigSettings(remoteConfigSettings)
  }
}
